public/stylesheets/index.css

```diff
@@ -51,7 +51,7 @@
   height: 40px;
   justify-content: flex-start;
   margin-right: 1.0px;
-  margin-top: 30px;
+  margin-top: 20px;
```

Comment:
Why is this changed?

Reply:
because we added a radio button above which changed the UX layout, it is from the design